config/locales/forms.it.yml

```diff
@@ -0,0 +1,169 @@
+it:
+  cancel: Annulla
+  formtastic:
+    labels:
+      tags_string: Tags
+      group:
+        name: Nome del gruppo
+        short_name: Sottodominio
+      group_request:
+        name: Name of the group
+        short_name: Web address
+        website: Your organisation's main website
+        email: Your organisation's main e-mail address
+        default_thread_privacy: Default privacy for discussion threads
+      group_membership:
+        role: Tipo di membership
+      group_pref:
+        notify_membership_requests: Enable membership request notifications
+      issue:
+        new: &id001
+          description: 59cc276c0ce886b3f7d1ca0eed363fb9_tr
+          location: 2cb1aa7c7a86a772f5e0cb21fcbc9971_tr
+          photo: 6b0d7090c503a0dafaf3a39d837ef4d8_tr
+          tags_string: 53f67be3879483cbcefa7f0e21fc5027_tr
+          title: f5f791770fc3a0079f829fd27ff3dba5_tr
+        create: *id001
+        edit: *id001
+        update: *id001
```

Comment:
Good spot - these are clearly wrong. :-( I'll investigate what's happened here - either a bug in pulling the translations, or perhaps garbage translations within transifex.